manifests/chain.pp

```diff
@@ -20,10 +21,11 @@
 define ferm::chain (
   Boolean $disable_conntrack,
   Boolean $log_dropped_packets,
-  String[1] $chain                     = $name,
-  Optional[Ferm::Policies] $policy     = undef,
-  Ferm::Tables $table                  = 'filter',
-  Array[Enum['ip','ip6']] $ip_versions = $ferm::ip_versions,
+  Boolean $drop_invalid_packets_with_conntrack = true,
```

Comment:
@foxxx0 I still think this is wrong. I would vote for setting this to false and disable_conntrack to true as default values. Afterwards set enable conntrack tracking for the input chain to true and do a major release.